File: samples/sample.cs
// AI generated sample

using System;
using System.Collections.Generic;

public class Bank {
    private string name;
    private Dictionary<int, Account> accounts;

    public Bank(string name) {
        this.name = name;
        this.accounts = new Dictionary<int, Account>();
    }

    public int CreateAccount(Customer customer, decimal initialDeposit) {
        Account account = new Account(customer, initialDeposit);
        accounts.Add(account.AccountNumber, account);
        return account.AccountNumber;
    }

    public void Deposit(int accountNumber, decimal amount) {
        if (accounts.TryGetValue(accountNumber, out Account account)) {
            account.Deposit(amount);
        } else {
            Console.WriteLine("Account not found.");
        }
    }

    public void Withdraw(int accountNumber, decimal amount) {
        if (accounts.TryGetValue(accountNumber, out Account account)) {
            account.Withdraw(amount);
        } else {
            Console.WriteLine("Account not found.");
        }
    }

    public void DisplayAccountDetails(int accountNumber) {
        if (accounts.TryGetValue(accountNumber, out Account account)) {
            account.DisplayDetails();
        } else {
            Console.WriteLine("Account not found.");
        }
    }
}

public class Account {
    private static int accountNumberCounter = 1000;
    public int AccountNumber { get; private set; }
    private Customer customer;
    private decimal balance;

    public Account(Customer customer, decimal initialDeposit) {
        this.customer = customer;
        this.balance = initialDeposit;
        this.AccountNumber = accountNumberCounter++;
    }

    public void Deposit(decimal amount) {
        balance += amount;
        Console.WriteLine($"Deposited {amount}. New balance: {balance}");
    }

    public void Withdraw(decimal amount) {
        if (amount > balance) {
            Console.WriteLine("Insufficient funds.");
        } else {
            balance -= amount;
            Console.WriteLine($"Withdrew {amount}. New balance: {balance}");
        }
    }

    public void DisplayDetails() {
        Console.WriteLine($"Account Number: {AccountNumber}");
        Console.WriteLine($"Customer Name: {customer.Name}");
        Console.WriteLine($"Account Balance: {balance}");
    }
}

public class Customer {
    public string Name { get; private set; }
    private string address;

    public Customer(string name, string address) {
        this.Name = name;
        this.address = address;
    }
}

// Example usage:
public class Program {
    public static void Main() {
        Bank bank = new Bank("Copilot Bank");

        // Create customers
        Customer john = new Customer("John Doe", "123 Elm Street");
        Customer jane = new Customer("Jane Doe", "456 Oak Street");

        // Create accounts
        int johnsAccount = bank.CreateAccount(john, 500);
        int janesAccount = bank.CreateAccount(jane, 1000);

        // Make some transactions
        bank.Deposit(johnsAccount, 300);
        bank.Withdraw(johnsAccount, 200);
        bank.Deposit(janesAccount, 500);
        bank.Withdraw(janesAccount, 1500);

        // Display account details
        bank.DisplayAccountDetails(johnsAccount);
        bank.DisplayAccountDetails(janesAccount);
    }
}

File: samples/sample.php
<?php

// AI generated sample

class Customer {
    private $name;
    private $address;

    public function __construct($name, $address) {
        $this->name = $name;
        $this->address = $address;
    }

    public function getName() {
        return $this->name;
    }
}

class Account {
    private static $accountNumberCounter = 1000;
    private $accountNumber;
    private $customer;
    private $balance;

    public function __construct($customer, $initialDeposit) {
        $this->customer = $customer;
        $this->balance = $initialDeposit;
        $this->accountNumber = self::$accountNumberCounter++;
    }

    public function deposit($amount) {
        $this->balance += $amount;
        echo "Deposited $amount. New balance: $this->balance\n";
    }

    public function withdraw($amount) {
        if ($amount > $this->balance) {
            echo "Insufficient funds.\n";
        } else {
            $this->balance -= $amount;
            echo "Withdrew $amount. New balance: $this->balance\n";
        }
    }

    public function displayDetails() {
        echo "Account Number: $this->accountNumber\n";
        echo "Customer Name: " . $this->customer->getName() . "\n";
        echo "Account Balance: $this->balance\n";
    }

    public function getAccountNumber() {
        return $this->accountNumber;
    }
}

class Bank {
    private $name;
    private $accounts;

    public function __construct($name) {
        $this->name = $name;
        $this->accounts = array();
    }

    public function createAccount($customer, $initialDeposit) {
        $account = new Account($customer, $initialDeposit);
        $this->accounts[$account->getAccountNumber()] = $account;
        return $account->getAccountNumber();
    }

    public function deposit($accountNumber, $amount) {
        if (isset($this->accounts[$accountNumber])) {
            $this->accounts[$accountNumber]->deposit($amount);
        } else {
            echo "Account not found.\n";
        }
    }

    public function withdraw($accountNumber, $amount) {
        if (isset($this->accounts[$accountNumber])) {
            $this->accounts[$accountNumber]->withdraw($amount);
        } else {
            echo "Account not found.\n";
        }
    }

    public function displayAccountDetails($accountNumber) {
        if (isset($this->accounts[$accountNumber])) {
            $this->accounts[$accountNumber]->displayDetails();
        } else {
            echo "Account not found.\n";
        }
    }
}

// Example usage:
$bank = new Bank("Copilot Bank");

// Create customers
$john = new Customer("John Doe", "123 Elm Street");
$jane = new Customer("Jane Doe", "456 Oak Street");

// Create accounts
$johnsAccount = $bank->createAccount($john, 500);
$janesAccount = $bank->createAccount($jane, 1000);

// Make some transactions
$bank->deposit($johnsAccount, 300);
$bank->withdraw($johnsAccount, 200);
$bank->deposit($janesAccount, 500);
$bank->withdraw($janesAccount, 1500);

// Display account details
$bank->displayAccountDetails($johnsAccount);
$bank->displayAccountDetails($janesAccount);
?>
